src/librustdoc/html/render/mod.rs

```diff
@@ -2747,6 +2751,7 @@ fn render_call_locations(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item) {
                   <summary class=\"hideme\">\
                      <span>More examples</span>\
                   </summary>\
+                  <div class=\"hide-more\">Hide additional examples</div>\
```

Comment:
Just realized: could you use `<details>` and `<summary>` to do this please? We use it for doc blocks and it has a few advantages like: no JS. :D

Reply:
That would also simplify your CSS since we already have the CSS for it.

Reply:
Can you explain a bit more what you mean? This UI is already using `<details>` and `<summary>`, so I'm not sure what you would like changed. The added line is text that shows up once you expand the `details`.

Reply:
Ah I see! Sorry I missed it. Then nevermind. :)